internal/makefile/makefile.go

```diff
@@ -375,13 +375,14 @@ endif
 			allSourceFilesExpr = `$(shell find -name *.rs)`
 		}
 
-		ignoreOptions := []string{}
+		var ignoreOptions []string
 		if cfg.GitHubWorkflow != nil {
 			for _, pattern := range cfg.GitHubWorkflow.License.IgnorePatterns {
 				// quoting avoids glob expansion
 				ignoreOptions = append(ignoreOptions, fmt.Sprintf("-ignore %q", pattern))
 			}
 		}
+		ignoreOptionsStr := strings.Join(append(ignoreOptions, "--"), " ")
```

Comment:
Why is the `--` mashed into the ignore options? Couldn't we `fmt.Sprintf("-c \"SAP SE\" %s -- %s")`?

Reply:
That generates two spaces before `--` if there are no ignore options. I find it likely that the space in front of `--` was there at some point and then removed to make the default case print nicely, so I tried phrasing it in a way that does not raise this urge again.